app/src/main/java/com/aptoide/uploader/account/Navigator.java

```diff
@@ -0,0 +1,9 @@
+package com.aptoide.uploader.account;
+
+public abstract class Navigator {
```

Comment:
Probably should be a LoginNavigator instead of Navigator. Navigator is too generic when this abstraction only deals with login views logic

Reply:
There is a LoginNavigator already that represents the navigation from the Login fragment. This class is only an abstract class that any other navigator can extend and isn't suppose to actually do any navigation